templates/default/less/variables.less

```diff
@@ -443,6 +444,17 @@
 @il-panel-listing-std-container-h4-color: #909090;
 @il-panel-listing-std-item-container-margin: 0 0 1px 0;
 
+//== Charts
+//
+//##
+
+@il-chart-item-active-bg: @brand-primary;
+@il-chart-item-active-color: white;
+@il-chart-item-inactive-bg: #e0e0e0;
+@il-chart-item-inactive-color: #909090;
+@il-chart-item-space: 2px;
```

Comment:
Thx for this good selection of less variables. If the colors concept will be accepted, I will try to work it in here.